impeller/aiks/picture.cc

```diff
@@ -3,11 +3,23 @@
 // found in the LICENSE file.
 
 #include "impeller/aiks/picture.h"
+#include <memory>
 
 #include "impeller/entity/entity.h"
+#include "impeller/renderer/render_target.h"
 
 namespace impeller {
 
-//
+std::shared_ptr<Image> Picture::RenderToImage(AiksContext& renderer) {
+  auto coverage = pass->GetElementsCoverage();
```

Comment:
Since this is a utility method, add validity checks. Some stuff that comes to mind:

* Null checks for pass.
* Check for empty coverage. You won't be able to create a zero sized offscreen texture anyway.

Reply:
Done. Added check for render target + coverage. Gotta be more careful about these coverage optionals -- I've forgotten to check them before.